Program.mk

```diff
@@ -16,22 +16,22 @@ endif
 #
 ifeq ("$(TOCK_BOARD)","hail")
 PROGRAM = $(TOCKLOADER) $(TOCKLOADER_GENERAL_FLAGS) install $<
-FLASH = $(TOCKLOADER) $(TOCKLOADER_GENERAL_FLAGS) install --jtag $<
+FLASH = $(TOCKLOADER) $(TOCKLOADER_GENERAL_FLAGS) install --jlink $<
 
 else ifeq ("$(TOCK_BOARD)","imix")
 # Change program region offset
 TOCKLOADER_INSTALL_FLAGS += --app-address 0x40000
```

Comment:
Does Imix still use a different address for apps too?

Reply:
Yes, although I think it most cases that flag is not required as the value is in the bootloader args.